releases/sui/2025-11-Sui-Role-Transfers-Release-v1.0.0.md

```diff
@@ -0,0 +1,238 @@
+# Sui Chains Role Transfers & Cap Management
+
+|                | **Owner**                              |
+| -------------- | -------------------------------------- |
+| **Created By** | @blockchainguyy <ayush@interoplabs.io> |
+| **Deployment** |                                        |
+
+| **Environment**      | **Chain** | **Deployment Status** | **Date** |
+| -------------------- | --------- | --------------------- | -------- |
+| **Devnet Amplifier** | `sui-2`   | -                     | TBD      |
+| **Stagenet**         | `sui`     | -                     | TBD      |
+| **Testnet**          | `sui`     | -                     | TBD      |
+| **Mainnet**          | `sui`     | -                     | TBD      |
+
+## Background
+
+Rotate non‑critical roles to appropriate operational addresses, and assign critical roles to EOA (ultimatly will be tranferred to Multisig/ServiceGovernance). This enforces correct permissions, separation of duties, and stronger security. 
+
+
+## Pre-requisites
+
+1. Sui CLI installed and configured for the target environment
+    ```bash
+    sui --version
+    sui client envs
+    ```
+2. Create an `.env` config for local commands
+    ```yaml
+    PRIVATE_KEY=<deployer private key or configured keystore alias>
+    ENV=<devnet|stagenet|testnet|mainnet>
+    CHAIN=sui
+    ```
+3. Ensure the following are present in `axelar-chains-config/info/${ENV}.json`
+    - `chains["sui"].contracts.AxelarGateway.{address, objects: {OwnerCap, UpgradeCap}}`
+    - `chains["sui"].contracts.GasService.{address,  objects: {OwnerCap, UpgradeCap, OperatorCap}}`
```

Comment:
I see we have steps to transfer `OwnerCap` and `UpgradeCap` but none for `OperatorCap`. Are we missing it in the doc? Or is it not needed